tests/src/whisk/core/cli/test/WskEntitlementTests.scala

```diff
@@ -86,7 +86,13 @@ class WskEntitlementTests
                 (wp, assetHelper) =>
                     assetHelper.withCleaner(wsk.action, fullyQualifiedActionName, confirmDelete = false) {
                         (action, name) =>
-                            val rr = action.create(name, None, shared = Some(true), update = true, expectedExitCode = FORBIDDEN)(wp)
+                            val rr = action.create(name, None, update = true, expectedExitCode = FORBIDDEN)(wp)
```

Comment:
could you please use var names that are more intuitive/less ambiguous? I'm guessing rr means "run result" and not "rodric rabbah". how about using "run" or "result"?